src/maven/pom.xml

```diff
@@ -46,7 +46,7 @@
   <pluginRepository>
    <id>mortbay-repo</id>
    <name>mortbay-repo</name>
-   <url>http://www.mortbay.org/maven2/snapshot</url>
+   <url>https://www.mortbay.org/maven2/snapshot</url>
```

Comment:
Dead repo

Reply:
'Allow edits from maintainers' is checked. Feel free to remove this.